pkg/apis/triggers/v1alpha1/event_listener_validation_test.go

```diff
@@ -260,11 +260,6 @@ func Test_EventListenerValidate(t *testing.T) {
 			},
 			Spec: v1alpha1.EventListenerSpec{
 				Triggers: []v1alpha1.EventListenerTrigger{{
-					Bindings: []*v1alpha1.EventListenerBinding{{
```

Comment:
Was this removal intentional?

Reply:
Yes because 
at a time user can specify either `triggerRef` or `triggerTemplate, triggerBinding, interceptors`

So removed `bindings` because there is `triggerRef`

Reply:
ahh so this was a bad test. Got it! Thanks!